### PR TITLE
Monthly confirmed subclass evidence update

### PR DESCRIPTION
**Pull request by twhetzel**
Update the confirmed subclass axiom annotation.

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -367554,6 +367554,7 @@ xref: SCTID:44423001 {source="MONDO:equivalentTo"}
 xref: UMLS:C0270855 {source="MEDGEN:124373", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0000412 {source="DOID:308"} ! neonatal period electroclinical syndrome
 is_a: MONDO:0005560 {source="DOID:308/inferred", source="EFO:1001900/inferred", source="NCIT:C116593"} ! brain disorder
+is_a: MONDO:0016801 {source="Orphanet:1935"} ! mitochondrial substrate carrier disorder
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:367557]: @twhetzel this SC has been excluded in Mondo (see line 367559) and therefore, it should not have been added. 
There is a bug in the pipeline which should be updated. 
How do you want to proceed? 
1) update the code and rerun this
2) manually remove this incorrect SC annotation, merge this PR, update the code in a separate PR before the next time we update the SC.

**Review by sabrinatoro — COMMENTED** [review]
@twhetzel 
please see my comment in line, and let me know how you would like to proceed.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (twhetzel) [reply src/ontology/mondo-edit.obo:367557]: For now I'd manually make the change to fix the incorrect SC annotation.

Comment (twhetzel) [reply src/ontology/mondo-edit.obo]: I have now removed this and submitted a ticket for [mondo-ingest](https://github.com/monarch-initiative/mondo-ingest/issues/680) about this.

**Review by sabrinatoro — APPROVED** [review]
looks good. Approving and Merging.

**Pull request merged**